### PR TITLE
DAOS-3127 build: RPM Build Files

### PR DESCRIPTION
**Pull request by ryon-jensen**
Signed-off-by: Ryon Jensen <ryon.jensen@intel.com>

### CODE REVIEW

**Review by brianjmurrell — CHANGES_REQUESTED** [review]
You seem to be missing the `debian/` subdir.

File: Dockerfile.ubuntu.18.04 [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+FROM ubuntu:18.04
+Maintainer John.E.Malmberg <John.E.Malmberg@intel.com>
+
+# Install basic tools
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+            autoconf bash curl debmake debhelper dpkg-dev doxygen gcc \
+            git locales make patch rpm
+
+# isa-l_crypto
+# isa-l_crypto needs a debhelper from backports
```
Comment (JohnMalmberg) [inline Dockerfile.ubuntu.18.04:16]: This was true for isa-l.  Is it true for isa-l_crypto?

**Review by JohnMalmberg — COMMENTED** [review]
If you want to have the Ubuntu packaging support after or test without it, you have to disable the builds in the jenkinsfile or the CI for this package will fail.

File: Dockerfile.ubuntu.18.04 (hunk 1 above)
Comment (ryon-jensen) [reply Dockerfile.ubuntu.18.04:16]: To be honest, I don't know. isa-l_crypto is very similar to isa-l, so most of this PR is just copy/pasted. I'm not sure why it was true for isa-l, so I'm not exactly sure how to go about finding out if it's true for isa-l_crypto. Any suggestions?

**Comment by ryon-jensen** [comment]
> You seem to be missing the `debian/` subdir.

Ya, I'm hoping to get it in without debian for now, and then I'll create a ticket to add debian support later. Debian packaging is a little out of my skill set currently so I will either request additional help to add debian or spend some background cycles learning debian packaging.

**Comment by brianjmurrell** [comment]
> I'll create a ticket to add debian support later.

That sounds fine.  Can you create that ticket and post the link here just to tie up the loose ends?

**Review by brianjmurrell — APPROVED** [review]

**Comment by ryon-jensen** [comment]
Ticket for debian packaging support: https://jira.hpdd.intel.com/browse/DAOS-3211

**Review by JohnMalmberg — APPROVED** [review]

**Pull request merged**